cpu/esp32/Makefile.dep

```diff
@@ -173,3 +173,10 @@ endif
 ifneq (,$(filter esp_jtag,$(USEMODULE)))
   FEATURES_REQUIRED += esp_jtag
 endif
+
+# enable bootloader reset over USB, requires CDC ACM to be used
+ifneq (,$(filter stdio_cdc_acm tinyusb_class_cdc,$(USEMODULE)))
```

Comment:
```suggestion
ifneq (,$(filter usbus_cdc_acm tinyusb_class_cdc,$(USEMODULE)))
```
As said, with https://github.com/RIOT-OS/RIOT/pull/19037, the `usb_board_reset` feature was moved to `usbus_cdc_acm` on STDIO. If any other implementation uses device class `usbus_cdc_acm`, the `usb_board_reset` is also required.

Reply:
tbh I think it would make sense if only `stdio_cdc_acm` could reset the board. If CDC ACM is used for something else we probably don't want the bootloader reset.

But changing it anyway for consistency.

Reply:
> If CDC ACM is used for something else we probably don't want the bootloader reset.

Why not? If you have a custom implementation that uses the CDC-ACM interface, it is still necessary to reset the card for flashing even if it is not used for STDIO. Board reset is a feature of any CDC ACM interface but not only the STDIO.

Reply:
I imagine you could have an application that acts as a simple USB <-> serial bridge. The USB is not the programming interface, you would program (and debug) the board using e.g. JTAG and you wouldn't want the board to accidentally enter bootloader mode just because you are doing something weird on the virtual COM port.

Reply:
Hm :thinking: good point. If you think it is better to leave it in the STDIO, we could revert PR #19037. However, my feeling the USB board reset function should not be bound to the STDIO but to the CDC ACM interface. This came up when I was implementing DFU for tinyUSB in PR #19010 for any reason, I don't remember exactly.